docs/uvcal_tutorial.rst

```diff
@@ -42,6 +42,23 @@ gives the beginning and end of the time range. The local sidereal times follow a
 pattern, UVCal objects should have either an ``lst_array`` or an ``lst_range`` attribute
 set.
 
+Generating calibration solutions typically requires choosing a convention concerning how
+polarized sky emission is mapped to the instrumental polarizations. For
+linear polarizations ``XX`` and ``YY``, the stokes ``I`` sky emission can be mapped to
+``I = (XX + YY)/2`` (the ``avg`` convention) or ``I = XX + YY`` (the ``sum``
+convention). This choice is generally encoded in the sky model to which the visibilities
+are calibrated. Different tools and simulators make different choices, generally following
+a standard choice for the field. In ``pyuvdata`` either of these choices are OK, but the
```

Comment:
> Different tools and simulators make different choices, generally following a standard choice for the field.

I think we need to be more explicit here about which tools do what, _especially_ if we are going to recommend that users always specify this parameter. In particular, I believe the tasks in CASA (e.g., tclean) and MIRIAD, along with WSClean, all assume the ``avg`` convention. I actually am not sure of what uses the ``sum`` convention, though I expect we can gather that information.

Reply:
I'm not very familiar with such tools, but I'll add that particular information.

Reply:
FHD uses the sum convention and the various HERA packages are moving to it.

Reply:
Cool. I added that as well.